src/app/containers/AudioVideo/index.stories.jsx

```diff
@@ -10,8 +13,18 @@ import {
   VideoClipGlobalWithoutCaption,
 } from './helpers/fixtureData';
 import AmpDecorator from '../../../../.storybook/helpers/ampDecorator';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+import AudioVideoContainer from '.';
 
 storiesOf('Audio Video Container', module)
+  .addDecorator(withKnobs)
+  .addDecorator(dirDecorator)
+  .add('story', ({ service }) => {
```

Comment:
Why do we need this?

Reply:
To be able to switch between diff services, unless it is not really necessary?

Reply:
On the storybook I see an empty story. It is no necessary to have it there.
<img width="1665" alt="Screenshot 2019-07-18 at 17 32 43" src="https://user-images.githubusercontent.com/48688870/61466208-2bf79800-a982-11e9-989b-8b20d25409ff.png">

Reply:
You need to update how fixture data is generated

https://github.com/bbc/simorgh/blob/8029cf8284866dc4a6e42b530b61e7b13da4ff9f/src/app/containers/ArticleFigure/fixtureData.jsx#L145-L148

Then create fixture data passing in `service` from `dirDecorator`
https://github.com/bbc/simorgh/blob/8029cf8284866dc4a6e42b530b61e7b13da4ff9f/src/app/containers/ArticleFigure/fixtureData.jsx#L209-L215

I hope this helps
https://github.com/bbc/simorgh/blob/8029cf8284866dc4a6e42b530b61e7b13da4ff9f/src/app/containers/ArticleFigure/index.stories.jsx#L23-L24